SeqPrep installation problem

Hi
I am trying to install SeqPrep but after make command this error appears and I am not figuring it out
Do you have any suggestions to overcome it? (I am using Linux mint maya 64bit)

cc  SeqPrep.o utils.o stdaln.o -lz -lm -o SeqPrep
SeqPrep.o: In function `main':
/home/pedro/Soft/SeqPrep-master/SeqPrep.c:350: undefined reference to `update_spinner'
collect2: ld returned 1 exit status
make: *** [SeqPrep] Error 1

kind regards,
Pedro Castro

Interesting. Which version of a c compiler are you using? If you have a specific version of gcc you use, you can compile with `make CC=gcc-4.X` for example.
> On Mar 30, 2015, at 5:33 PM, pcastro88 <notifications@github.com> wrote:
> 
> Hi
> I am trying to install SeqPrep but after make command this error appears and I am not figuring it out
> Do you have any suggestions to overcome it? (I am using Linux mint maya 64bit)
> 
> cc SeqPrep.o utils.o stdaln.o -lz -lm -o SeqPrep
> SeqPrep.o: In function main':
> /home/pedro/Soft/SeqPrep-master/SeqPrep.c:350: undefined reference toupdate_spinner'
> collect2: ld returned 1 exit status
> make: *** [SeqPrep] Error 1
> 
> kind regards,
> Pedro Castro
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/jstjohn/SeqPrep/issues/24>.
>

--- a/SeqPrep.c
+++ b/SeqPrep.c
@@ -79,33 +79,19 @@ void help ( char *prog_name ) {
   exit( 1 );
 }
 
-static unsigned short spcount = 0;
 /**
  * Have a nice spinner to give you a false sense of hope
  */
-extern inline void update_spinner(unsigned long long num_reads){
+static inline void update_spinner(unsigned long long num_reads){
+  static unsigned short spcount = 0;
+
   if(num_reads == 0){
     fprintf(stderr,"Processing reads... |");
     fflush(stderr);
   }else if (num_reads % SPIN_INTERVAL == 0){
-    switch(spcount % 4){
-    case 0:
-      fprintf(stderr,"\b/");
-      fflush(stderr);
-      break;
-    case 1:
-      fprintf(stderr,"\b-");
-      fflush(stderr);
-      break;
-    case 2:
-      fprintf(stderr,"\b\\");
-      fflush(stderr);
-      break;
-    default:
-      fprintf(stderr,"\b|");
-      fflush(stderr);
-      break;
-    }
+    putc('\b',stderr);
+    putc("/-\\|"[spcount % 4],stderr);
+    fflush(stderr);
     spcount++;
   }
 }

